.github/workflows/lint-php.yml

```
@@ -35,12 +39,16 @@ jobs:
   lint-php:
     timeout-minutes: 5
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        php-version: ${{ fromJson(inputs.PHP_MATRIX) }}
     env:
       COMPOSER_AUTH: '${{ secrets.COMPOSER_AUTH_JSON }}'
+    strategy:
+      matrix:
+        php-version: ${{ inputs.PHP_MATRIX && fromJSON(inputs.PHP_MATRIX) || fromJson('["custom"]') }}
```

Can we move this below `runs-on` again?
```suggestion
    strategy:
      matrix:
        php-version: ${{ inputs.PHP_MATRIX && fromJSON(inputs.PHP_MATRIX) || fromJson('["custom"]') }}
    env:
      COMPOSER_AUTH: '${{ secrets.COMPOSER_AUTH_JSON }}'
```

---

Should be fine 👍🏻 the `strategy.matrix` has no access to `env` anyway.